wire-compiler/src/main/java/com/squareup/wire/model/ProtoTypeName.java

```diff
@@ -45,14 +45,24 @@
   public static final ProtoTypeName UINT32 = new ProtoTypeName("uint32");
   public static final ProtoTypeName UINT64 = new ProtoTypeName("uint64");
 
+  public static final ProtoTypeName FILE_OPTIONS = get("google.protobuf", "FileOptions");
+  public static final ProtoTypeName MESSAGE_OPTIONS = get("google.protobuf", "MessageOptions");
+  public static final ProtoTypeName FIELD_OPTIONS = get("google.protobuf", "FieldOptions");
+  public static final ProtoTypeName ENUM_OPTIONS = get("google.protobuf", "EnumOptions");
+  public static final ProtoTypeName ENUM_VALUE_OPTIONS = get("google.protobuf", "EnumValueOptions");
+  public static final ProtoTypeName SERVICE_OPTIONS = get("google.protobuf", "ServiceOptions");
+  public static final ProtoTypeName METHOD_OPTIONS = get("google.protobuf", "MethodOptions");
```

Comment:
Can do package scope. I started out thinking of this entire thing as an implementation detail; but I think it's going to be the thing that plugins are going to want to read.